Tests/SourceKittenFrameworkTests/SwiftDocMarkupDelimiterTests.swift

```diff
@@ -0,0 +1,352 @@
+//
+//  SwiftDocMarkupDelimiterTests.swift
+//  SourceKittenFrameworkTests
+//
+//  Created by Eneko Alonso on 5/28/20.
+//  Copyright © 2020 SourceKitten. All rights reserved.
+//
+
+import SourceKittenFramework
+import XCTest
+
+let markupDelmitersXML = """
```

Comment:
Maybe this could be a file in the `Fixtures/` directory instead?

Reply:
I wasn't sure how fixtures are set up, but sounds like a good idea.